app/ui/src/main/java/com/fsck/k9/activity/compose/AttachmentPresenter.java

```diff
@@ -110,16 +120,45 @@ private boolean hasLoadingAttachments() {
         return false;
     }
 
-    public ArrayList<Attachment> createAttachmentList() {
+    public ArrayList<Attachment> createAttachmentListWithoutResizing() {
```

Comment:
Please keep the method's original name. You forgot to update at least one caller (`MessageCompose.draftIsNotEmpty()`) and now the wrong method is called.